lms/templates/logout.html

```diff
@@ -0,0 +1,23 @@
+{% extends "main_django.html" %}
```

Comment:
Thanks @robrap. I have removed usage of `escapejs` for `target`. The call to `trans` is fine. It's just a call to `gettext`. Additionally, that text is hard-coded.